examples/scripts/iframe/index.mustache

```diff
@@ -66,16 +66,12 @@
         function loadEngine(callback) {
             const enginePath = urlParams && urlParams.get('use_local_engine') || "{{{enginePath}}}";
 
-            fetch(enginePath)
-                .then(function(response) { return response.text() })
-                .then(function(data) {
-                    var module = {
-                        exports: {}
-                    };
-                    window.pc = (Function('module', 'exports', data).call(module, module, module.exports), module).exports;
-                    window.top.pc = window.pc;
```

Comment:
Can we retain this line so that `pc` is accessible from the console when viewing the examples browser?

Reply:
Yeah sure.

Just fyi: if you select the example iframe in the "Javascript context" dropdown in Chrome's console tab you can also access pc there.